include/curl/curl.h

```diff
@@ -748,6 +748,31 @@ typedef CURLcode (*curl_ssl_ctx_callback)(CURL *curl,    /* easy handle */
                                                           mbedtls_ssl_config */
                                           void *userptr);
 
+struct ssl_session_dump {
```

Comment:
Why `dump` ? shouldn't it be something about session id?

And a public struct name needs to be prefixed with `curl_`.

Reply:
`curl_ssl_sessionid_dump` ?